named-tunnel-k8s/cloudflared.yaml

```diff
@@ -75,6 +76,8 @@ data:
     credentials-file: /etc/cloudflared/creds/credentials.json
     # Serves the metrics server under /metrics and the readiness server under /ready
     metrics: 0.0.0.0:2000
+    # Autoupdates applied in a k8s pod will be lost when the pod is removed or restarted, so
+    # autoupdate doesn't make sense in Kubernetes. You should really set this outside Kubernetes though.
```

Comment:
> You should really set this outside Kubernetes though.

I'd make it slightly less informal:

"We recommend `no-autoupdate: false` when running outside Kubernetes though".